tests/unit/source_code/test_notebook.py

```diff
@@ -0,0 +1,297 @@
+from collections.abc import Callable
+
+import pytest
+from databricks.sdk.service.workspace import Language
+
+from databricks.labs.ucx.source_code.notebook import Notebook, NotebookDependencyGraph
+from tests.unit import _load_sources
+
+PYTHON_NOTEBOOK_SAMPLE = (
+    "00_var_context.py.sample",
+    Language.PYTHON,
+    ['md', 'md', 'md', 'python', 'python', 'python', 'md', 'python', 'md'],
+)
+PYTHON_NOTEBOOK_WITH_RUN_SAMPLE = (
+    "01_var_market_etl.py.sample",
+    Language.PYTHON,
+    [
+        'md',
+        'run',
+        'md',
+        'python',
+        'md',
+        'python',
+        'python',
+        'python',
+        'python',
+        'md',
+        'python',
+        'python',
+        'md',
+        'python',
+        'python',
+        'python',
+        'md',
+        'python',
+        'md',
+        'python',
+        'python',
+        'md',
+        'python',
+    ],
+)
+SCALA_NOTEBOOK_SAMPLE = (
+    "01_HL7Streaming.scala",
+    Language.SCALA,
+    [
+        'md',
+        'md',
+        'scala',
+        'sql',
+        'md',
+        'scala',
+        'scala',
+        'md',
+        'md',
+        'scala',
+        'md',
+        'scala',
+        'sql',
+        'sql',
+        'md',
+        'scala',
+        'md',
+        'scala',
+        'sql',
+        'sql',
+        'sql',
+        'sql',
+        'sql',
+        'sql',
+        'sql',
+        'md',
+        'scala',
+        'md',
+        'md',
+    ],
+)
+R_NOTEBOOK_SAMPLE = (
+    "3_SparkR_Fine Grained Demand Forecasting.r",
+    Language.R,
+    [
+        'md',
+        'md',
+        'md',
+        'r',
```

Comment:
looks too long and noisy as for the test sample

Reply:
maybe we can move these to json files ?

Reply:
I'm not alone, see https://github.com/psf/black/issues/1281
Fixed by disabling `fmt` for these lines